scripts/raspberryconfig.sh

```diff
@@ -110,27 +110,52 @@ wget http://repo.volumio.org/Volumio2/wireless-firmwares/brcmfmac43143.bin -P /l
 #dpkg -i /raspi/raspi-config_20151019_all.deb
 #rm -Rf /raspi
 
-echo "Installing WiringPi"
-wget http://repo.volumio.org/Volumio2/Binaries/wiringpi_2.24_armhf.deb
-dpkg -i wiringpi_2.24_armhf.deb
-rm /wiringpi_2.24_armhf.deb
+echo "Installing WiringPi from Raspberrypi.org Repo"
+apt-get -y install wiringpi
```

Comment:
I think `wiringpi` is a very-well supported component, and availability in `raspberrypi.org` repo is very safe for longterm (mostly similar confidence than with Raspbian) : this should ensure maximum compatibility & bug-fixes warranty.
I consider being lucky to have spotted issue with specific volumio repo`wiringpi_2.24` and new kernel. The more we lock-in to specific versions in volumio repo, the more we get exposed to such upgrade issues I think.
We may want to only have very specific tweaked software in volumio repo: the fewer there are, the more manageable it is.